### PR TITLE
CardNudge: RTL issue for swiping left/right

### PR DESCRIPTION
**Pull request by Anubhvv**
### Problem 
Swiping animation was not following the direction of swipe action (left or right).
The background "hide" text was not aligned correctly in RTL
### Root cause 
ReverseDirection was defaulting to "false" in swipeable.
Also, positive/ negative value for "text".
### Validations

(how the change was tested, including both manual and automated tests)

### Screenshots

| Before                                                        |  After                                                      |
|----------------------------------------------|--------------------------------------------|
|  | |

### Pull request checklist

This PR has considered:
- [ ] Light and Dark appearances
- [ ] Automated Tests
- [ ] Documentation and demo app examples
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and RTL layouts
- [ ] Size classes and window sizes (notched devices, multitasking, different window sizes, etc)

### CODE REVIEW

File: fluentui_notification/src/main/java/com/microsoft/fluentui/tokenized/notification/CardNudge.kt [hunk 1]
```diff
@@ -130,9 +134,15 @@ fun CardNudge(
         ) {
             LaunchedEffect(state.offset.value) {
                 if (state.offset.value > 0.1F) {
-                    metadata.rightSwipeGesture?.invoke(state.offset.value / maxWidth)
+                    when {
+                        isRtl -> metadata.rightSwipeGesture?.invoke(-state.offset.value / maxWidth) //negative value for RTL for aligning the "Hide" Text appropriately
+                        else -> metadata.rightSwipeGesture?.invoke(state.offset.value / maxWidth)
```
Comment (mishramayank1) [inline fluentui_notification/src/main/java/com/microsoft/fluentui/tokenized/notification/CardNudge.kt:139]: Revert this change. It is consumer responsibility to process it as per layout direction.

File: fluentui_notification/src/main/java/com/microsoft/fluentui/tokenized/notification/CardNudge.kt [hunk 2]
```diff
@@ -130,9 +134,15 @@ fun CardNudge(
...
+                    }
                 } else if (state.offset.value < -0.1F) {
-                    metadata.leftSwipeGesture?.invoke(state.offset.value / maxWidth)
+                    when {
```
Comment (NamanPandey) [inline fluentui_notification/src/main/java/com/microsoft/fluentui/tokenized/notification/CardNudge.kt:142]: same as above

File: fluentui_notification/src/main/java/com/microsoft/fluentui/tokenized/notification/CardNudge.kt (hunk 1 above)
Comment (Anubhvv) [reply fluentui_notification/src/main/java/com/microsoft/fluentui/tokenized/notification/CardNudge.kt]: Done,
also  fixed this: (Now it shows "Left Swiped: some value"
Screenshot_2023-08-30-15-23-13-45_138c6ebfedc8e8a8cd1938476bab217a

**Review by mishramayank1 — APPROVED** [review]
